templates/csv.yaml

```diff
@@ -83,7 +83,7 @@ spec:
 
     The components provided with the OpenShift Serverless operator require minimum cluster sizes on
     OpenShift Container Platform. For more information, see the documentation on [Getting started
-    with OpenShift Serverless](https://access.redhat.com/documentation/en-us/openshift_container_platform/__OCP_TARGET__/html-single/serverless/index#serverless-getting-started).
+    with OpenShift Serverless](https://access.redhat.com/documentation/en-us/openshift_container_platform/__OCP_TARGET__/html/serverless/index#install-serverless-operator).
```

Comment:
Did you mean

https://access.redhat.com/documentation/en-us/openshift_container_platform/__OCP_TARGET__/html/serverless/install ?

(there's no install-serverless-operator anchor on the index page, https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/serverless/index#install-serverless-operator  open just the root of the index, same as  https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/serverless/index page )

Reply:
The link should be https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/serverless/install#install-serverless-operator